package-lock.json

```diff
@@ -7661,7 +7661,7 @@
     },
     "external-editor": {
       "version": "2.2.0",
-      "resolved": "https://registry.npmjs.org/external-editor/-/external-editor-2.2.0.tgz",
+      "resolved": "http://registry.npmjs.org/external-editor/-/external-editor-2.2.0.tgz",
```

Comment:
make sure these do not get rolled to http

Reply:
gotcha